src/integration_tests/schemas/test_integration_unhappy_path.py

```diff
@@ -0,0 +1,170 @@
+    def test_get_schema_validation_error(self):
+        """
+        Test validation issue by providing an invalid survey_id for GET /v1/schema.
+        * Expected: 400 Bad Request.
+        """
+        response = self.session.get(
+            f"{config.API_URL}/v1/schema",
+            headers=self.headers,
+        )
+        assert response.status_code == 400
+
+    @pytest.mark.order(6)
+    def test_get_schema_metadata_unauthorized(self):
+        """
+        Test unauthorized access by providing incorrect authorization token for GET /v1/schema_metadata.
+        * Expected: 401 Unauthorized.
+        """
+        response = self.session.get(
+            f"{config.API_URL}/v1/schema_metadata?survey_id={test_survey_id}",
+            headers=self.invalid_token_headers,
+        )
+        assert response.status_code == 401
+
+    @pytest.mark.order(7)
+    def test_get_schema_metadata_validation_error(self):
+        """
+        Test validation issue by providing an invalid survey_id for GET /v1/schema_metadata.
+        * Expected: 400 Bad Request.
+        """
+        response = self.session.get(
+            f"{config.API_URL}/v1/schema_metadata",
+            headers=self.headers,
+        )
+        assert response.status_code == 400
+
+    @pytest.mark.order(8)
+    def test_get_schema_metadata_404_not_found(self):
+        """
+        Test data not found by requesting nonexistent schema metadata for GET /v1/schema_metadata.
+        * Expected: 404 Not Found.
+        """
+        response = self.session.get(
+            f"{config.API_URL}/v1/schema_metadata?survey_id={self.invalid_survey_id}",
+            headers=self.headers,
+        )
+        assert response.status_code == 404
+
+    @pytest.mark.order(9)
+    def test_get_schema_v2_unauthorized(self):
+        """
+        Test unauthorized access by providing incorrect authorization token for GET /v2/schema.
+        * Expected: 401 Unauthorized.
+        """
+        response = self.session.get(
+            f"{config.API_URL}/v2/schema?guid=invalid_guid",
+            headers=self.invalid_token_headers,
+        )
+        assert response.status_code == 401
+
+    @pytest.mark.order(10)
+    def test_get_schema_v2_validation_error(self):
+        """
+        Test validation issue by providing an invalid GUID for GET /v2/schema.
+        * Expected: 400 Bad Request.
+        """
+        response = self.session.get(
+            f"{config.API_URL}/v2/schema",
+            headers=self.headers,
+        )
+        assert response.status_code == 400
+
+    @pytest.mark.order(11)
+    def test_get_schema_v2_404_not_found(self):
+        """
+        Test data not found by requesting a nonexistent GUID for GET /v2/schema.
+        * Expected: 404 Not Found.
+        """
+        response = self.session.get(
+            f"{config.API_URL}/v2/schema?guid=nonexistent_guid",
+            headers=self.headers,
+        )
+        assert response.status_code == 404
```

Comment:
could you add tests using nonsense in params e.g. /v2/schema?tfyghjkohjghfghgjhkjlghhj